### PR TITLE
Added invitation announcement blog post for PAAP research project

### PR DESCRIPTION
**Pull request by dstengle**
This is the announcement calling for interviewees for the platform research.

### CODE REVIEW

**Comment by dstengle** [comment]
@mluskind @Rotfuks Please have a look.

File: website/content/en/blog/announce-paap-research.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+The interview process itself is pretty simple and consists of a 45 minute conversation with a couple of interviewers. Hopefully we can get multiple people from the same organization to participate between different roles like platform engineers, internal platform users and platform leaders.
+
+All interview data is kept confidential and will only be used after being anonymized during our analysis across interviews.
+
+Our goal is to have some initial analysis to talk about for the upcoming Platform Day at KubeCon NA in Salt Lake City.
+
+So if you're interested in participating in this research, please [sign up here](https://bit.ly/platform-research).
+
+We also you to [join the CNCF Slack](https://communityinviter.com/apps/cloud-native/cncf) and find us in the #wg-platforms channel. Working group meetings are every two weeks and participation is open to all!
```
Comment (abangser) [inline website/content/en/blog/announce-paap-research.md:33]: Small typo at the start of this?

**Review by abangser — COMMENTED** [review]
Very good stuff! Excited to get your work more visibility 🚀

Just a small typo comment and hoping to see reviews from someone else in the paap group but overall looking great 🙌

File: website/content/en/blog/announce-paap-research.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+## (Don't worry, you don't need to be a product manager!)
+
+The CNCF Platforms Working Group is pleased to announce a research effort to increase our understanding of how the industry is building internal platforms and we'd like you to participate.
+
+After the publishing of the Platform Maturity model, the working group's next area of focus has been on Platform As A Product. This approach treats our users as customers of the platform, which is a product that should address their needs.  While there is certainly a lot of coverage on this topic in conferences and public forums, it is unclear how widespread these practices are. 
+
+While the name of the research project might imply that we're only interested in talking to product managers, that definitely isn't the case! We want to talk to anyone building or using platforms to find out how teams decide what to build for their users.
+
+So if you build platforms but don't think of yourself as a product manager or your organization doesn't do product management, we'd still love to talk to you.
```
[the remark this answers is not in the document]
Comment (mluskind) [reply website/content/en/blog/announce-paap-research.md:23]: @dstengle @Rotfuks that's the impression that I got as well. Update looks good. Can we remove "using platforms." They're a good check on what builders tell us but most likely wont be able to answer most of the core questions around practise within the team. Better to get a referral to consumers from platform builders.

File: website/content/en/blog/announce-paap-research.md (hunk 1 above)
Comment (dstengle) [reply website/content/en/blog/announce-paap-research.md:33]: Thanks for the catch!

File: website/content/en/blog/announce-paap-research.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+- Platform As A Product
+---
+
+# Invitation to participate in Platform As A Product Research 
+
+## (Don't worry, you don't need to be a product manager!)
+
+The CNCF Platforms Working Group is pleased to announce a research effort to increase our understanding of how the industry is building internal platforms and we'd like you to participate.
+
+After the publishing of the Platform Maturity model, the working group's next area of focus has been on Platform As A Product. This approach treats our users as customers of the platform, which is a product that should address their needs.  While there is certainly a lot of coverage on this topic in conferences and public forums, it is unclear how widespread these practices are.
```
[the remark this answers is not in the document]
Comment (dstengle) [reply website/content/en/blog/announce-paap-research.md:19]: All caps is best.

File: website/content/en/blog/announce-paap-research.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dstengle) [reply website/content/en/blog/announce-paap-research.md:23]: Updated to this for next commit:

"We want to talk to anyone building platforms to find out how teams decide what to build for their users. So regardless of whether you see yourself or your organization as doing product management - we'd love to talk to you!"

File: website/content/en/blog/announce-paap-research.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dstengle) [reply website/content/en/blog/announce-paap-research.md:33]: The slack link doesn't work unless you are already

**Comment by dstengle** [comment]
@abangser I think this is ready to go!

**Comment by dstengle** [comment]
> @abangser I think this is ready to go!

Should I create new PR with the changes and include the signed-off line in the commit to get DCO to pass?

**Comment by dstengle** [comment]
Due to my messing up git yet again I may need to resubmit this PR from a clean fork anyway.

**Pull request closed without merging**